app/moves/middleware.js

```diff
@@ -22,13 +22,18 @@ function makeMultipleRequests(service, dateRange, locationIdBatches) {
 module.exports = {
   redirectBaseUrl: (req, res) => {
     const today = format(new Date(), dateFormat)
-    const currentLocation = get(req.session, 'currentLocation.id')
-
-    if (currentLocation) {
-      return res.redirect(`${req.baseUrl}/day/${today}/${currentLocation}`)
+    const currentLocation = get(req.session, 'currentLocation')
+
+    switch (currentLocation && currentLocation.location_type) {
```

Comment:
I don't think this logic is right at this stage. I think we need to keep the current logic of if a current location is set redirect to a location. 

And maybe in this instance we need to use permissions to determine the next path. So if a user has permission to view proposed moves, they are directed there?

Reply:
This logic would leave STC/SCH users at a dead end I think.

Reply:
so something like   . ?:

     const today = format(new Date(), dateFormat)
    const currentLocation = get(req.session, 'currentLocation.id')

    if (currentLocation) {
      const userPermissions = get(req.session, 'user.permissions')
      const canViewProposedMoves = permissions.check('move:proposed:view', userPermissions)
      return res.redirect(
        `${req.baseUrl}/day/${today}/${currentLocation}/${canViewProposedMoves ? 'proposed' : ''}`
      )
    }

    return res.redirect(`${req.baseUrl}/day/${today}`)

Reply:
Yeah, I think that feels a little safe for now 👍

Reply:
Once we introduce a "dashboard" at this route we can get rid of this anyway